src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -102,6 +97,26 @@ $table-cell-height-half: $table-cell-height / 2;
   width: 15 * $baseline-grid;
 }
 
+.kd-replicasetdetail-tab-header {
+  md-tabs-canvas,
+  md-pagination-wrapper {
+    height: $toolbar-height-size-base;
+  }
+
+  md-tab-item {
+    padding-top: $toolbar-height-size-base / 2;
+    text-transform: none;
+  }
+
+  md-tab-content {
+    &.md-left,
+    &.md-right {
+      animation: none;
```

Comment:
I've looked a bit more and found a better solution. `$animate.enabled()` doesn't seem to be working.

https://github.com/angular/material/issues/811#issuecomment-65164436

This looks to be a good way to do it. Do you think we can disable it globally for tabs?